frontend/src/components/ResizeTableWrapper/ResizeTableWrapper.tsx

```diff
@@ -0,0 +1,39 @@
+import { ColumnsType } from 'antd/lib/table';
+import React, { useState } from 'react';
+import { ResizeCallbackData } from 'react-resizable';
+
+function ResizeTableWrapper(props: ResizeTableWrapperProps): JSX.Element {
+	const { columns, children } = props;
+	const [columnsData, setColumns] = useState<ColumnsType>(columns);
+
+	const handleResize = (index: number) => (
+		_e: React.SyntheticEvent<Element>,
+		{ size }: ResizeCallbackData,
+	): void => {
+		const newColumns = [...columnsData];
```

Comment:
would it possible to move the logic in utils ?

Reply:
It is wrapped in useCallback & wrapper file so, its fine to have it on file.